src/docs/user/references/Variables.adoc

```diff
@@ -141,22 +141,25 @@ Examples: PA:REGEXP([a-z]+) will accept "abc", "foo", but not "Foo".
 
 . Advanced variable models
 
-* *PA:CATALOG_OBJECT*, *PA:CATALOG_OBJECT(kind)*, *PA:CATALOG_OBJECT(,contentType)*, *PA:CATALOG_OBJECT(,,bucketName)*, *PA:CATALOG_OBJECT(,,,objectName)*, *PA:CATALOG_OBJECT(kind,contentType,bucketName,objectName)* :
+* *PA:CATALOG_OBJECT* :
 use this type when you want to reference and use, in the current workflow, another object from the Catalog. Various portals and tools will make it easy to manage catalog objects. For instance, at Job submission, you will be able to browse the Catalog to select the needed object. +
-Variable syntax must be a valid expression that matches the following pattern: `bucketName/objectName[/revision]`.
+Variable value syntax must be a valid expression that matches the following pattern: `bucketName/objectName[/revision]`.
 Note that the revision sub-pattern is a hash code number represented by 13 digit. +
-A filter can be specified in the PA:CATALOG_OBJECT model. In that case, the variable value must be a catalog object which matches the kind, content type, bucketName and objectName requirements. Note that kind and content type are case insensitive and require a "startsWith" matching, while bucketName and objectName are case sensitive and, by default, require a "contains" matching.
-That is, the kind and the content type of the provided catalog object must start with the filters specified in the model while bucketName and objectName of the catalog object should contain them.
-The scheduler server verifies that the object exists in the catalog and fulfills the requirements when the workflow is submitted. +
+Examples: PA:CATALOG_OBJECT will accept "bucket-example/object-example" and "bucket-example/object-example/1539310165443" but not "bucket-example/object-example/153931016" (invalid revision number) nor "bucket-example/" (missing object name).
+
+* *PA:CATALOG_OBJECT(kind,contentType,bucketName,objectName)* :
```

Comment:
You show a single syntax, but I think we should display all syntaxes, no ?

Reply:
Well in the first sentence I added : `One or more filters can be specified`. And in the examples I handle these cases.
It is clearer if we put only the generic full syntax and after more details.

Reply:
I suggest

One or more filters can be specified in the specified order of the PA:CATALOG_OBJECT model to limit the accepted values. For example PA:CATALOG_OBJECT(kind) can be used to filter a specific kind, PA:CATALOG_OBJECT(kind,contentType) to filter both kind and content-type,  PA:CATALOG_OBJECT(,contentType) to filter only a content-type (note the empty first parameter), etc...